tests/test_dataset_deterministic.py

```diff
@@ -0,0 +1,122 @@
+import pytest
+import argparse
+import random
+import os
+import webdataset as wds
+import glob
+from open_lm.model import _MODEL_CONFIGS
+from open_lm.main import random_seed
+from open_lm.data import get_wds_dataset
+from open_lm.file_utils import get_string_for_epoch, get_metadata_file, get_shards_for_chunk
+from open_lm.params import parse_args
+from pathlib import Path
+
+NUM_SAMPLES = 100000
+
+# Update this to two data sources with webdataset, each with their own manifest.
+INPUT_PATHS = [
```

Comment:
any way we can create a `/tests/assets` folder and put a small amount of mock data there for testing rather than pointing to local paths?

Reply:
for this also probably good if we can check in some of our val data as opposed to rpj or pile shards

Reply:
Added a ```/tests/assets``` with a few tarfiles.

Reply:
For now I just added some of our training shards, lmk if we want to change that.